pepys_admin/view_data_cli.py

```diff
@@ -87,13 +90,21 @@ def do_view_table(self):
         """Asks user to select a table name. Converts table name to class name,
         fetches the objects up to the number of MAX_ROWS_DISPLAYED, and prints them in table format.
         """
-        headers = list()
-        associated_attributes = list()
         table_names = self._get_table_names()
         message = "Select a table >"
         selected_table = iterfzf(table_names, prompt=message)
         if selected_table is None:
             return
+
+        text = self.generate_view_table_text(selected_table)
+        print(text)
```

Comment:
@robintw - is the above a debug line?

Reply:
Not really - it's in there to ensure that if we get a failure of the test, we actually know which table it failed on. Ideally we'd use a 'parameterised' pytest but we can't do that as we need to generate the list of tables based on access to the DataStore object.

It could be removed, but then whenever the test failed we'd have no idea what table it failed on without adding this line back in.

Reply:
Sure, ok.